feat(blog): add bpmn-js v16 blog post

philippfromme opened the pull request:
Related to https://github.com/bpmn-io/internal-docs/issues/869
Related to https://github.com/bpmn-io/internal-docs/issues/861

nikku commented:
Added a bunch of review hints via https://github.com/bpmn-io/bpmn.io/pull/178/commits/cb98a91960a4fc46d92d2d94f931c38973aebc6c:

* Title is title case
* Our pattern is `XYZ ships with SOME_TOOL`
* Ensure the screenshots link to the demo
* Prominently link bpmn-js (the source)

nikku reviewed (commented):
Looks good to me.

nikku commented:
Additional thought: Consider removing the vertical rendering (we can feature that with a quick screencapture via our social channels), and combine it with the dmn-js latest major: https://github.com/bpmn-io/dmn-js/blob/develop/packages/dmn-js/CHANGELOG.md#1500. In both we migrated ("we migrated full stack").

nikku reviewed (dismissed):
Let's rework this slightly according to https://github.com/bpmn-io/bpmn.io/pull/178#issuecomment-1854196066.

philippfromme commented:
@nikku I have rewritten the post to feature both bpmn-js and dmn-js. I also removed the vertical pool rendering support so the post is focused on the updated labels/translations. Please have a look.

nikku commented on src/blog/2023/008-bpmn-js-16-0-0.md:
```diff
@@ -0,0 +1,51 @@
+---
+
+title: Better Labels
+description: "The bpmn-js@16 and dmn-js@15 releases ships updated translations."
```
```suggestion
description: "The bpmn-js@16 and dmn-js@15 releases ship with improved labels. This may impact your custom translations."
```

nikku commented on src/blog/2023/008-bpmn-js-16-0-0.md:
```diff
@@ -0,0 +1,51 @@
+description: "The bpmn-js@16 and dmn-js@15 releases ships updated translations."
+preview_image: assets/attachments/blog/2023/008-translations.png
+layout: blogpost.hbs
+slug: 2023-bpmn-js-16-0-0-translations
```
```suggestion
slug: 2023-better-labels
```

nikku commented on src/blog/2023/008-bpmn-js-16-0-0.md:
```diff
@@ -0,0 +1,51 @@
+---
+
+title: Better Labels
```
```suggestion
title: Consistent, Better Labels
```

nikku reviewed (commented):
Great stuff. Let's rename the file according to the new content and I think we're good.

philippfromme commented:
@nikku I updated the screenshot. Should be ready to be merged after stashing the commits.

nikku commented on src/blog/2023/008-bpmn-js-16-0-0.md:
```diff
@@ -0,0 +1,51 @@
+
+Did we miss anything? Did you spot a bug, or would you like to suggest an improvement?
+
+Reach out to us via [our forums](https://forum.bpmn.io/), tweet us [@bpmn_io](https://twitter.com/bpmn_io), or file an issue you found in [the bpmn-js issue tracker](https://github.com/bpmn-io/bpmn-js/issues).
```
```suggestion
Reach out to us via [our forums](https://forum.bpmn.io/), [follow us on Mastodon](https://fosstodon.org/@bpmn_io), and file an issue you find in the respective project issue trackers.
```

nikku commented on src/blog/2023/008-bpmn-js-16-0-0.md:
```diff
@@ -0,0 +1,51 @@
+
+Reach out to us via [our forums](https://forum.bpmn.io/), tweet us [@bpmn_io](https://twitter.com/bpmn_io), or file an issue you found in [the bpmn-js issue tracker](https://github.com/bpmn-io/bpmn-js/issues).
+
+Get the latest BPMN modeling toolkit pre-packaged or as source code via [npm](https://www.npmjs.com/package/bpmn-js) or [unpkg](https://unpkg.com/bpmn-js/).
```
```suggestion
```

nikku commented on src/blog/2023/008-bpmn-js-16-0-0.md:
```diff
@@ -0,0 +1,51 @@
+
+Did we miss anything? Did you spot a bug, or would you like to suggest an improvement?
+
+Reach out to us via [our forums](https://forum.bpmn.io/), [follow us on Mastodon](https://fosstodon.org/@bpmn_io), and file an issue you find in the respective project issue trackers.
```
Cf. https://github.com/bpmn-io/bpmn.io/pull/181

nikku reviewed (commented):
Great stuff :clap:
